mathics/builtin/numbers/randomnumbers.py

```diff
@@ -145,9 +143,11 @@ def randchoice(self, n, size, replace, p):
 
 class RandomState(Builtin):
     """
+    <url>:WMA: https://reference.wolfram.com/language/ref/RandomState.html</url>)
```

Comment:
Should not be a colon before the closing URL tag?

Reply:
There is a stray ")" at the end. That should be fixed now.